src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -1416,8 +1414,7 @@ private void inviteFocus(final EntityBareJid roomIdentifier)
             StanzaCollector collector = null;
             try
             {
-                collector = ((ProtocolProviderServiceJabberImpl) xmppProvider)
-                    .getConnection()
+                collector = getConnection()
```

Comment:
can this fail with NPE here if no connection?

Reply:
maybe it doesn't  matter I dont know.

Reply:
Yep, it can, but I don't see how this is possible to try inviting the focus you will get connected. And I just changed to use the getConnection() without changing the logic.

Reply:
yes